mantidimaging/core/utility/cuda_check.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (C) 2020 ISIS Rutherford Appleton Laboratory UKRI
+# SPDX - License - Identifier: GPL-3.0-or-later
+
+import os
+from typing import Tuple
+
+
+def _read_from_terminal(command: str) -> str:
+    """
+    Runs a terminal command and returns the result.
+    """
+    return os.popen(command).read()
```

Comment:
I think a better built-in to use is [subprocess.check_output](https://docs.python.org/3/library/subprocess.html#subprocess.check_output)